lib/improver/tests/optical_flow/test_AdvectField.py

```diff
@@ -252,8 +268,14 @@ def test_background_values(self):
 
     def test_time_step(self):
         """Test outputs are OK for a time step with non-second components"""
-        expected_output = np.zeros(shape=(4, 3))
-        result = self.plugin.process(self.cube, datetime.timedelta(hours=1))
+        vel_x = self.plugin.vel_x.copy(data=self.plugin.vel_x.data / 6.)
+        vel_y = self.plugin.vel_y.copy(data=self.plugin.vel_y.data / 6.)
```

Comment:
You can modify the initialised plugin directly:
self.plugin.vel_y.data = self.plugin.vel_y.data / 6.
Saves creating a new plugin object.